quantum/quantum_keycodes.h

```diff
@@ -116,6 +116,11 @@ enum quantum_keycodes {
     KC_ASON,   // 5C1B
     KC_ASOFF,  // 5C1C
 
+    // Tapping term setup
+    TK_PRNT,
+    TK_UP,
+    TK_DOWN,
+
```

Comment:
Looking at this again -- given that the comments above and below have hex values, it's likely this section will need to be relocated to the end?

Reply:
Solved in https://github.com/qmk/qmk_firmware/pull/11036/commits/470d50ee39d034af086a33f2aa9fc7d1d3e0a977